### DISCUSSION TITLE
Provide more !FREE! domains and if it possible one free subdomain

### DISCUSSION THREAD

**Discussion by AsikLy** [simple-login/app#1599] [Feature Requests]
Please give free users more domains on which you can create aliases. I would also like you to return the domain SimpleLogin.com or provide the opportunity to create one free subdomain. At the moment I don't see any advantages of your service compared to its counterparts. You provide very limited functionality at the moment.

**Comment by nguyenkims** [comment]
This is duplicated to this discussion https://github.com/simple-login/app/discussions/1589
